src/main/java/io/quarkus/gizmo2/creator/BlockCreator.java

```diff
@@ -1,6 +1,6 @@
 package io.quarkus.gizmo2.creator;
 
-import static java.lang.constant.ConstantDescs.CD_boolean;
+import static java.lang.constant.ConstantDescs.*;
```

Comment:
I think that we should not use the `*` imports. In fact, gizmo should probably use the same [rules as Quarkus](https://github.com/quarkusio/quarkus/blob/main/independent-projects/ide-config/pom.xml).

Reply:
I agree for class imports. For static imports, especially for classes like this which are designed to be used in this way, I disagree. It seems pointless to expand them all out in this case.